galaxy/management/commands/generate_galaxy_test_data.py

```diff
@@ -0,0 +1,288 @@
+        def _tag_neighbors(uid, neighbor_dist, pict_power, pict_dist):
+            u2 = self.users[uid - neighbor_dist]
+            u3 = self.users[uid + neighbor_dist]
+            PeoplePictureRelation(user=u2, picture=self.picts[uid**pict_power]).save()
+            PeoplePictureRelation(user=u3, picture=self.picts[uid**pict_power]).save()
+            PeoplePictureRelation(user=u2, picture=self.picts[uid - pict_dist]).save()
+            PeoplePictureRelation(user=u3, picture=self.picts[uid - pict_dist]).save()
+            PeoplePictureRelation(user=u2, picture=self.picts[uid + pict_dist]).save()
+            PeoplePictureRelation(user=u3, picture=self.picts[uid + pict_dist]).save()
+
+        for uid in range(200, self.NB_USERS - 200):
+            u1 = self.users[uid]
+            self.logger.info(f"Pictures of {u1}")
+            PeoplePictureRelation(user=u1, picture=self.picts[uid]).save()
+            PeoplePictureRelation(user=u1, picture=self.picts[uid - 14]).save()
+            PeoplePictureRelation(user=u1, picture=self.picts[uid + 14]).save()
+            PeoplePictureRelation(user=u1, picture=self.picts[uid - 20]).save()
+            PeoplePictureRelation(user=u1, picture=self.picts[uid + 20]).save()
+            PeoplePictureRelation(user=u1, picture=self.picts[uid - 21]).save()
+            PeoplePictureRelation(user=u1, picture=self.picts[uid + 21]).save()
+            PeoplePictureRelation(user=u1, picture=self.picts[uid - 22]).save()
+            PeoplePictureRelation(user=u1, picture=self.picts[uid + 22]).save()
+            PeoplePictureRelation(user=u1, picture=self.picts[uid - 30]).save()
+            PeoplePictureRelation(user=u1, picture=self.picts[uid + 30]).save()
+            PeoplePictureRelation(user=u1, picture=self.picts[uid - 31]).save()
+            PeoplePictureRelation(user=u1, picture=self.picts[uid + 31]).save()
+            PeoplePictureRelation(user=u1, picture=self.picts[uid - 32]).save()
+            PeoplePictureRelation(user=u1, picture=self.picts[uid + 32]).save()
+
+            if uid % 3 == 0:
+                _tag_neighbors(uid, 1, 1, 40)
+            if uid % 5 == 0:
+                _tag_neighbors(uid, 2, 1, 50)
+            if uid % 10 == 0:
+                _tag_neighbors(uid, 3, 1, 60)
+            if uid % 20 == 0:
+                _tag_neighbors(uid, 5, 1, 70)
+            if uid % 25 == 0:
+                _tag_neighbors(uid, 10, 1, 80)
+
+            if uid % 2 == 1:
+                _tag_neighbors(uid, 1, 2, 90)
+            if uid % 15 == 0:
+                _tag_neighbors(uid, 5, 2, 100)
+            if uid % 30 == 0:
+                _tag_neighbors(uid, 4, 2, 110)
+
+    def make_important_citizen(self, uid):
+        # These three people will be really close, since they already are in many clubs together!
+        u1 = self.users[uid]
+        u2 = self.users[uid - 100]
+        u3 = self.users[uid + 100]
+        u1.godfathers.add(u3)
+        u1.godchildren.add(u2)
```

Comment:
```suggestion
        u1.godfathers.add(u2, u3)
```

Reply:
Nope, I want a godfather and a godchild for `u1`

Reply:
I had misread it. My bad.